ui/raidboss/data/06-ew/trial/golbez-ex.ts

```diff
@@ -170,7 +170,7 @@ const triggerSet: TriggerSet<Data> = {
           en: 'Role positions',
           de: 'Rollenposition',
           fr: 'Positions par rôle',
-          ja: 'ロール特定位置へ',
+          ja: '4:4あたまわり',
```

Comment:
I'm sorry to bother you, but is this definitely not an error?
ご迷惑をおかけして申し訳ありませんが、これは確かにエラーではないですか？

Reply:
I'm sorry. I cannot remember exactly, but I think I translated the English part "Role Positions" into Japanese verbatim.
It is long time ago since i have played Golbez Trial, so I'm not sure if that is "4:4 healer stacks".

Would you please modify if it is incorrect. 😊

Reply:
Understood, I will proceed through [#5811](https://github.com/quisquous/cactbot/pull/5811).
かしこまりました、[#5811](https://github.com/quisquous/cactbot/pull/5811)を通じて進めさせていただきます。